pallets/simple-nft/src/mock.rs

```diff
@@ -65,6 +65,17 @@ parameter_types! {
     pub const MaxMetadataCount: u32 = 3;
 }
 
+#[derive(Encode, Decode, Clone, PartialEq, Debug, Eq, Ord, PartialOrd)]
+pub enum Role {
+    Admin,
+}
```

Comment:
I think we should have a test using multiple roles.

Reply:
I've added another Role to this mock enum + added a test where more than one role is set.

Reply:
@jonmattgray I only see one test referring to the new role `NotAdmin` (`it_fails_for_destroying_single_token_as_incorrect_role`) which is clearly a failure case. Am I missing something?

Reply:
you're right I'll add a happy path one for multiple roles

Reply:
new test added